File: eNote/UI/enote_desktop/lib/models/adresa.dart
import 'package:json_annotation/json_annotation.dart';

part 'adresa.g.dart';

@JsonSerializable()
class Adresa {
  int? id;
  String? grad;
  String? ulica;
  String? broj;

  Adresa();

  factory Adresa.fromJson(Map<String, dynamic> json) => _$AdresaFromJson(json);

  Map<String, dynamic> toJson() => _$AdresaToJson(this);
}

File: eNote/eNote.Model/DTOs/Adresa.cs
namespace eNote.Model.DTOs
{
    public class Adresa
    {
        public int Id { get; set; } 
        public string Grad { get; set; } = null!;
        public string Ulica { get; set; } = null!;
        public string Broj { get; set; } = null!; 
    }
}
